controllers/contacts.js

```diff
@@ -0,0 +1,60 @@
+const {
+    listContacts,
+    getContactById,
+    removeContact,
+    addContact,
+    updateContact,
+} = require('../models/contacts');
+
+const { HttpError, ctrlWrapper } = require('../helpers');
+
+const getAll = async (req, res) => {
```

Comment:
```suggestion
const getAllContact = async (req, res) => {
```
Аналогічно для інших контроллерів